src/utils/vault-provider.ts

```diff
@@ -637,11 +644,13 @@ export default class VaultProvider {
         if (coin.balance) {
           switch (coin.ticker) {
             case TickerKey.CACAO:
-              return api.coin
-                .coingeckoValue(coin.ticker, currency)
-                .then((value) => {
-                  coin.value = value;
-                });
+              return true
+                ? api.coin
+                    .mayachainValue()
+                    .then((value) => (coin.value = value))
+                : api.coin
+                    .coingeckoValue(coin.ticker, currency)
+                    .then((value) => (coin.value = value));
```

Comment:
We are using  Mayachain Midgard as temporary  source for price of cacao because coingecko is returning invalid value  and we should revert it back in few days. So this is just a temporary approach.